src/metrics/generated/proto/metricpb/metric.proto

```diff
@@ -34,16 +34,19 @@ enum MetricType {
 message Counter {
   bytes id = 1;
   int64 value = 2;
+  bytes annotation = 3;
 }
 
 message BatchTimer {
   bytes id = 1;
   repeated double values = 2;
+  bytes annotation = 3;
 }
 
 message Gauge {
   bytes id = 1;
   double value = 2;
+  bytes annotation = 3;
 }
 
```

Comment:
if you are changing protos, make sure to update `src/metrics/encoding/protobuf/reset.go` to reuse the new fields, too.